Package.swift

```diff
@@ -1,5 +1,13 @@
 import PackageDescription
 
 let package = Package(
-  name: "Quick"
+    name: "Quick",
+    targets: [
+        Target(name: "QuickTests", dependencies: [.Target(name: "Quick"), .Target(name: "QuickTestHelpers")]),
+        Target(name: "QuickFocusedTests", dependencies: [.Target(name: "Quick"), .Target(name: "QuickTestHelpers")]),
+        Target(name: "QuickTestHelpers", dependencies: [.Target(name: "Quick")]),
+    ],
+    dependencies: [ // TODO: This ought to be `testDependencies:` but there seems to be a swiftpm bug causing the lib to not get linked then
```

Comment:
I'm actually thinking now that this isn't really a bug, but rather has to do with the `swift test` command not having been built out yet. Presumably test dependencies shouldn't be linked when building the package itself, only its tests, however swiftpm hasn't been properly taught about test targets yet. I'll update the comment accordingly.